pom.xml

```diff
@@ -3,7 +3,7 @@
     <parent>
         <groupId>org.jenkins-ci.plugins</groupId>
         <artifactId>plugin</artifactId>
-        <version>1.609.2</version><!-- which version of Jenkins is this plugin built against? -->
+        <version>2.6</version>
```

Comment:
A lower versión could be used, indeed.

Neverthless, I'm currently using a snapshot with this PR on a Jenkins 2.4
and is working (so there seems no API changes has been made).

Will try on Monday to lower that parent, rebuild and retest.

El viernes, 20 de mayo de 2016, Jason Swager <notifications@github.com>
escribió:

> In pom.xml
> <https://github.com/jenkinsci/vsphere-cloud-plugin/pull/45#discussion_r64046348>
> :
>
> > @@ -3,7 +3,7 @@
> >      <parent>
> >          <groupId>org.jenkins-ci.plugins</groupId>
> >          <artifactId>plugin</artifactId>
> > -        <version>1.609.2</version><!-- which version of Jenkins is this plugin built against? -->
> > +        <version>2.6</version>
>
> Is this necessary? This will require every user to be Jenkins 2.6 - not
> yet released. And not everyone is upgrading to the very latest and greatest
> immediately. Can a lower version be safely used?
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub
> <https://github.com/jenkinsci/vsphere-cloud-plugin/pull/45/files/4e1c73896ef6ea27afc2e0053005e69b3a50f9d5#r64046348>
>


-- 
*It* is *better* to die() than to return() in failure